debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -442,6 +452,10 @@ public boolean isReadOnlyDatabaseConnection() {
         return readOnlyDatabaseConnection;
     }
 
+    public boolean isSkipLowActivityLSNsEnabled() {
```

Comment:
Tiny nit: "LSN" -> "Lsn" (this isn't done consistently across the code base yet, but it's the casing we want to converge on eventually for abbreviations in type/method/variable names).